stages/org.osbuild.ostree.deploy.container

```diff
@@ -92,7 +91,7 @@ def make_fs_identifier(desc):
 def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
     images = containers.parse_containers_input(inputs)
     for image in images.values():
-        with containers.container_source(image) as (_, image_source):
+        with containers.container_source(image) as (__, image_source):
```

Comment:
Oh the use of double underscore was mentioned during one of the classes I'm taking in order to ignore values you won't actually be using (which is why I changed it from a single to double underscore). However, I probably should have fact-checked what the professor said since a single underscore should suffice when you need to ignore a value: https://www.datacamp.com/tutorial/role-underscore-python#2.-ignoring-values-%3Cstro. Will change it back to a single underscore.

Reply:
I still see this in there - along with the other changes in this file that aren't significant.